R/peer-review-badge.R

```diff
@@ -0,0 +1,95 @@
+#' Check whether README.md features an rOpenSci software-review badge
+#'
+#' @inheritParams srr_counts
+#'
+#' @return A string, empty if the badge was found.
+#' @export
+#'
+readme_badge <- function(repourl, repo, issue_id, post_to_issue = TRUE) {
+  # Content taken directly from editor_check():
+  branch <- roreviewapi::get_branch_from_url (repourl)
```

Comment:
should this content actually become a function then?

Reply:
the first part of the code is used in several functions so should it be refactored

Reply:
Oh, i get you. Yeah, it is a problem that. I think the process represented by all those can't conveniently be factored out into any other function, but yes, the actual content of that line should be. I'll do that right now; thanks!

Reply:
Oh no, wait! The problem is that that line must also be called directly from `plumber.R`, and plumber has no access to internal package functions. So factoring that off to a separate function would also require that function to be exported. Exported a trivial and otherwise entirely useless function like that would likely be little more than confusing, and that's why i opted for just repeating same code over and over. Unless you've got any other brilliant suggestions?

Reply:
no, that sounds good now that I understand the constraints better!